demos/Agent_Based_Modeling/prey_predator_abm/model/config.py

```diff
@@ -1,21 +1,22 @@
 from cadCAD.configuration import Experiment
 from cadCAD.configuration.utils import config_sim
-from .state_variables import genesis_states
-from .partial_state_update_block import partial_state_update_block
-from .sys_params import sys_params 
+from demos.Agent_Based_Modeling.prey_predator_abm.model.state_variables import genesis_states
+from demos.Agent_Based_Modeling.prey_predator_abm.model.partial_state_update_block import partial_state_update_block
+from demos.Agent_Based_Modeling.prey_predator_abm.model.sys_params import sys_params
 
 sim_config = config_sim (
     {
         'N': 3, # number of monte carlo runs
-        'T': range(300), # number of timesteps
+        # 'T': range(300), # number of timesteps
+        'T': range(10), # number of timesteps
         'M': sys_params, # simulation parameters
     }
 )
 
 exp = Experiment()
-
-exp.append_configs(
+exp.append_model(
```

Comment:
`Experiment().append_model` is new name for `Experiment().append_configs` but is backwards compatible